Fabric.Authorization.API/appsettings.json

```diff
@@ -46,5 +46,6 @@
   },
   "EntityFrameworkSettings": {
     "MinimumLogLevel": "Warning"
-  }
+  },
+  "MigrateDuplicateGroups": true

```

Comment:
Should we also add a default to the web.config, that way we can tell a user to flip it to false if necessary.

Reply:
Yes